source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/chemistry/Chemistry.java

```diff
@@ -0,0 +1,17 @@
+package nl.overheid.aerius.shared.domain.chemistry;
```

Comment:
license header

Reply:
Move this enum to `OPSOptions` class. It should not be a separate package in `domain` as this is a small feature and that doesn't justify a package on that level.